.github/workflows/playwright.yml

```diff
@@ -7,30 +7,39 @@ env:
   HUSKY: 0
 
 jobs:
-  playwright-test:
-    name: Test
+  Run:
+    name: Shard ${{ matrix.shardIndex }}
     if: github.event.deployment_status.state == 'success' && github.event.deployment_status.environment == 'Preview'
     timeout-minutes: 60
     runs-on: ubuntu-latest
+    env:
```

Comment:
PW was taking way too long — went to a sharded set up, which was made easier with the new `merge-report` util in playwright